crates/goose-mcp/src/google_drive/mod.rs

```diff
@@ -1343,68 +1357,105 @@ impl GoogleDriveRouter {
                     "The fileId param is required".to_string(),
                 ))?;
 
-        // extract pageSize, and convert it to an i32, default to 10
+        let comment_id = params.get("commentId").and_then(|q| q.as_str());
+
+        // extract pageSize, and convert it to an i32, default to 20
         let page_size: i32 = params
             .get("pageSize")
             .map(|s| {
                 s.as_i64()
                     .and_then(|n| i32::try_from(n).ok())
                     .ok_or_else(|| ToolError::InvalidParameters(format!("Invalid pageSize: {}", s)))
                     .and_then(|n| {
-                        if (0..=100).contains(&n) {
+                        if (1..=100).contains(&n) {
```

Comment:
i think we can expand the max to something like 500, since its all text information it shouldn't be too big of a deal context wise